lib/propshaft/helper.rb

```diff
@@ -4,22 +4,40 @@ def compute_asset_path(path, options = {})
       Rails.application.assets.resolver.resolve(path) || raise(MissingAssetError.new(path))
     end
 
-    # Add an option to call `stylesheet_link_tag` with `:all` to include every css file found on the load path.
+    # Add an option to call `stylesheet_link_tag` with
+    # `:all` to include every css file found on the load path or
+    # `:app` to include every css file found on `app/assets/stylesheets`.
     def stylesheet_link_tag(*sources, **options)
-      if sources.first == :all
+      case sources.first
+      when :all
         super(*all_stylesheets_paths, **options)
+      when :app
+        super(*app_stylesheets_paths, **options)
       else
         super
       end
     end
 
-    # Returns a sorted and unique array of logical paths for all stylesheets in the load path.
-    def all_stylesheets_paths
-      Rails.application.assets.load_path
-        .assets(content_types: [ Mime::EXTENSION_LOOKUP["css"] ])
-        .collect { |css| css.logical_path.to_s }
-        .sort
-        .uniq
-    end
+    private
+      def all_stylesheets_paths
+        stylesheets_paths_for(Rails.application.assets.load_path)
+      end
+
+      def app_stylesheets_paths
+        stylesheets_paths_for(
+          Propshaft::LoadPath.new(
+            [ File.join(Rails.root, "app", "assets", "stylesheets") ],
+            compilers: Rails.application.assets.compilers,
+            version: Rails.application.config.assets.version))
```

Comment:
Think we can do this with `Rails.application.assets.load_path.dup.tap { |l| l.paths = [ File.join(Rails.root, "app", "assets", "stylesheets") ] }` or something like that. So you don't have to repeat the configuration.

Reply:
Actually I see there's not a setter on paths. We can just add that, but it'll need to clear the cache when set.

Reply:
I did it, although I had to add a setter to clear the cache.

Reply:
> Actually I see there's not a setter on paths. We can just add that, but it'll need to clear the cache when set.

Ah yes, I just read this